test/docs/__snapshots__/get-docs-yaml.api-test.ts.snap

```diff
@@ -143,6 +143,39 @@ paths:
           description: Customer not found.
       tags:
         - customers
+  /api/v1/emails:
```

Comment:
Remember to regenerate after making changes to `emails.controller.ts`.

Reply:
Done